### PR TITLE
refactor(FX-4394): Reorganize artwork sections (artwork page redesigne phase 2)

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves [FX-4394] 

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [x] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Reorganize artwork sections for artwork page redesigne phase 2 - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Reorganize artwork sections (artwork page redesigne phase 2) - dimatretyak

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-4394]: https://artsyproduct.atlassian.net/browse/FX-4394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  We are trying to migrate away from moment towards `luxon`, but found moment imports in the following new files:

- `src/app/Scenes/Artwork/Components/ArtworkLotTimer.tsx`

See [docs](https://moment.github.io/luxon/api-docs/index.html).
  
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 440d47638b66b0065bc5a4daccb34e91e4f710da
</p>

File: src/app/Scenes/Artwork/Artwork.tsx [hunk 1]
```diff
@@ -337,10 +321,161 @@ export const Artwork: React.FC<ArtworkProps> = ({
...
+      sections.push({
+        key: "otherWorks",
+        element: <OtherWorks artwork={artworkBelowTheFold} />,
+      })
+    }
+
+    return sections
+  }
+
+  const redesignedSectionsData = (): ArtworkPageSection[] => {
```
Comment (gkartalis) [inline src/app/Scenes/Artwork/Artwork.tsx:398]: I ❤️ this approach!

File: src/app/Scenes/Artwork/Artwork.tsx [hunk 2]
```diff
@@ -467,22 +603,41 @@ export const Artwork: React.FC<ArtworkProps> = ({
...
+                      </Box>
+                    )
+                  }
+
+                  return (
+                    <Box mx={2}>
+                      <Separator />
+                    </Box>
+                  )
+                }}
```
Comment (gkartalis) [inline src/app/Scenes/Artwork/Artwork.tsx:627]: 👏

**Review by gkartalis — APPROVED** [review]
great refactor! Love it! 🎉 will do a quick QA on my device as well but consider this approved

**Pull request merged**